sites/canonical-com.yaml

```diff
@@ -0,0 +1,28 @@
+name: canonical-com
+domain: canonical.com
+
+container:
+  image: prod-comms.docker-registry.canonical.com/canonical.com
+
+useSquid: true
```

Comment:
This means using the `squid.internal` proxy settings, right?

This should really be the default for all sites, and do I think ideally it wouldn't have a setting - it would just be applied. The only thing is, if we change this to make it default for all sites then we have to be careful to roll out these changes one site at a time to make sure they don't break anything. Still I think that's worth doing.

Would you mind removing this option and simply making it so these settings are applied for all sites?

Reply:
Renamed to useProxy as we agreed.

Reply:
Filed https://github.com/canonical-web-and-design/deployment-configs/issues/280 for removing this option, after this is merged.

Reply:
Thank you Robin!